samples/bot-adaptive-card-actions/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-bot-adaptivecard-actions-nodejs",
+    "source": "officeDev",
+    "title": "Get Adaptivecard Actions",
+    "shortDescription": "Demonstrating the feature where user can send adaptivecard actions using bot.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-adaptivecard-actions/nodejs",
+    "longDescription": [
+      "Demonstrating the feature where user can send adaptivecard actions using bot."
```

Comment:
Demonstrating the feature where user can send adaptive card with different actions using bot.